ci/get-unifi.sh

```diff
@@ -2,27 +2,37 @@
 # Copyright (c) 2025 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 set -euo pipefail
+TEMP_DIR="$(mktemp -d)"
+ORAS_VERSION="1.2.2"
+ORAS="oras"
 # Unifi latest url
-UNIFI_URL="${1:-europe-docker.pkg.dev/da-images-dev/oci-playground/components/assistant:latest}"
+UNIFI_URL="${1:-europe-docker.pkg.dev/da-images-dev/oci-private/components/assistant:latest}"
```

Comment:
It looks like both places where we're running get-unifi.sh, we supply this argument - I think it'd be better to remove the default value here and just error out when the argument isn't supplied.